modules/ROOT/pages/mule-oauth-provider-landing-page.adoc

```diff
@@ -119,25 +119,22 @@ You can define scopes in three different places:
 ...
 ----
 
-Full https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform] asset available on Exchange.
+For more information, see https://anypoint.mulesoft.com/exchange/org.mule.templates/api-gateway-oauth2-provider/[Mule OAuth 2.0 server for Anypoint Platform].
 
 == Client Application Cache
 
-To prevent downtime of the Mule OAuth 2.0 provider due to errors when connecting to the platform,
-the Mule OAuth Client Store caches each valid Client Application for which a token is requested.
+To prevent any possible downtime of the Mule OAuth 2.0 provider, the Mule OAuth Client Store caches each valid Client Application for which a token is requested. The downtime can occur due to errors when connecting to Anypoint Platform.
```

Comment:
Changes the meaning. It is not any possible downtime, but specific ones, as stated originally.  The details at the end, negate the first statement in the corrected version.

Reply:
Fixed!